server/src/util/event-email.ts

```diff
@@ -325,6 +327,38 @@ export const eventAttendanceConfirmation = ({
   );
 };
 
+interface WaitlistConfirmation {
+  event: { name: string };
+  userName: string;
+}
+
+export const eventWaitlistConfirmation = ({
+  event,
+  userName,
+}: WaitlistConfirmation) =>
+  withUnsubscribe(
+    eventWaitlistConfirmationText({
+      eventName: event.name,
+      userName: userName ? ` ${userName}` : '',
```

Comment:
the type is always `true`, and they will be used when there is a user, the if statement can go, unless I am missing something

```suggestion
      userName: ` ${userName}`,
```

Reply:
Nah, it can be empty, _anonymous_ is only displayed in client. Resolver takes user directly from db, or from ctx, both don't have empty string replaced with _anonymous_.

As a side note - I'd be a bit surprised if page on which I didn't give anything as my name would start email with _Hi anonymous_ :slightly_smiling_face: